content/source/docs/enterprise-beta/api/run.html.md

```diff
@@ -59,17 +62,37 @@ curl \
 }
 ```
 
+## Upload Configuration files
 
-## Create a Run on the Workspace
+| Method | Path           |
+| :----- | :------------- |
+| POST | The upload URL is provided in the `upload-url` attribute in the `configuration-versions` resource |
+
+### Parameters
+
+- `data` (`file: <required>`) - A local .tar.gz file containing the folder of the terraform configuration files.
+
+### Sample Request
 
-**Method**: POST
+```shell
+$ curl \
+    --request POST \
+    -F 'data=@config.tar.gz` \
+    http://127.0.0.1:7675/v1/object/4c44d964-eba7-4dd5-ad29-1ece7b99e8da
+```
 
-**Path**: /runs
+
+## Create a Run on the Workspace
+A Run performs a Plan and Apply on the last configuration version created and using the variables set in the workspace.
+
+| Method | Path           |
+| :----- | :------------- |
+| POST | /runs |
```

Comment:
this is probably better as: `/workspaces/:workspace_id/runs`, and then just don't include the relationship info in the payload